### ISSUE TITLE
add platform Ubuntu touch

### ISSUE THREAD

**Issue by maddin200** [fyne-io/fyne-cross#165]
aim for the OpenStore from Ubuntu touch

**Comment by Jacalz** [comment]
Please fill out the issue templates when opening issues. They are intended to make it easier for us to categorise and handle issues.

I don’t think there is anything to add to support this. We already support Linux compilation and the only real difference is that you want to pass -tags mobile (maybe “mobile,wayland” if they require that) to fyne-cross. This should already be doable as far as I know.

**Comment by Bluebugs** [comment]
Isn't there some packaging requirement to generate an application for the Ubuntu store?

**Comment by Jacalz** [comment]
Submitting on https://open-store.io/ perhaps?

**Comment by Bluebugs** [comment]
I was expecting that Ubuntu touch relied on snap format, but I can not find any documentation.

**Comment by maddin200** [comment]
Thanks.
I think the stuff relies on the clickable format:
https://clickable-ut.dev/en/latest/
or
https://ubports.com/de/blog/ubports-blogs-nachrichten-1/post/introduction-to-clickable-147

**Comment by andydotxyz** [comment]
As @Jacalz says this a packaging format question rather than a new platform? Like "arch" or "flatpak".
So you can indeed take a Fyne-cross compiled app and package it up appropriately.

In the future we should consider different distributions for the result of packaging process, I think that would satisfy this need?

**Comment by Jacalz** [comment]
I think this would make more sense over in fyne for producing metadata for the formats rather than here in fyne-cross. Most binary format distributions do not allow, or strongly prefer, you to compile the binaries using their build systems. Just like Flatpak only really want binary only packages in cases where the app is proprietary, I doubt the other formats would allow us to cross compile packages/binaries for them to distribute. The best we can do is probably just to produce the required metadata and build scripts for loading into their build/packaging solution.

**Comment by Bluebugs** [comment]
I agree with @Jacalz especially now that we have a PR to rely on fyne for build and that the long term plan is to merge fyne-cross into fyne CLI.

**Comment by andydotxyz** [comment]
Agreed. Of course it may be that we get it in "Fyne-cross" or "Fyne cross" for free in the future as @Bluebugs indicates it is coming together.

**Comment by lucor** [comment]
Indeed, support to additional platforms in the future would be nice to have.
